README.md

```diff
@@ -48,95 +59,56 @@ The script will generate solar forecasts at the specified intervals between the
+
 ```bash
 pip install quartz-solar-forecast
 ```
+
 You might need to install the following packages first
+
 ```bash
 conda install -c conda-forge pyresample
-````
-This can solve the [bug: ___kmpc_for_static_fini](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/issues/32).
-
-## Model
-
-The model is a gradient boosted tree model and uses 9 NWP variables.
-It is trained on 25,000 PV sites with over 5 years of PV history, which is available [here](https://huggingface.co/datasets/openclimatefix/uk_pv).
-The training of this model is handled in [pv-site-prediction](https://github.com/openclimatefix/pv-site-prediction)
-TODO - we need to benchmark this forecast. 
-
-The 9 NWP variables, from Open-Meteo documentation, are mentioned above with their appropariate units. 
-
-1. **Visibility (km)**, or vis: Distance at which objects can be clearly seen. Can affect the amount of sunlight reaching solar panels.
-2. **Wind Speed at 10 meters (km/h)**, or si10 : Wind speed measured at a height of 10 meters above ground level. Important for understanding weather conditions and potential impacts on solar panels.
-3. **Temperature at 2 meters (°C)**, or t : Air temperature measure at 2 meters above the ground. Can affect the efficiency of PV systems. 
-4. **Precipiration (mm)**, or prate : Precipitation (rain, snow, sleet, etc.). Helps to predict cloud cover and potentiel reductions in solar irradiance. 
-5. **Shortwave Radiation (W/m²)**, or dswrf: Solar radiation in the shortwave spectrum reaching the Earth's surface. Measure of the potential solar energy available for PV systems. 
-6. **Direct Radiation (W/m²)** or dlwrf: Longwave (infrared) radiation emitted by the Earth back into the atmosphere. **confirm it is correct**
-7. **Cloud Cover low (%)**, or lcc: Percentage of the sky covered by clouds at low altitudes. Impacts the amount of solar radiation reachign the ground, and similarly the PV system.
-8. **Cloud Cover mid (%)**, or mcc : Percentage of the sky covered by clouds at mid altitudes. 
-9. **Cloud Cover high (%)**, or lcc : Percentage of the sky covered by clouds at high altitude
-
-We also use the following features
-- poa_global: The plane of array irradiance, which is the amount of solar radiation that strikes a solar panel.
-- poa_global_now_is_zero: A boolean variable that is true if the poa_global is zero at the current time. This is used to help the model learn that the PV generation is zero at night.
-- capacity (kw): The capacity of the PV system in kw.
-- The model also has a feature to check if these variables are NaNs or not.
-
-The model also uses the following variables, which are currently all set to nan
-- recent_power: The mean power over the last 30 minutes
-- h_mean: The mean of the recent pv data over the last 7 days
-- h_median: The median of the recent pv data over the last 7 days
-- h_max: The max of the recent pv data over the last 7 days
-
-## Known restrictions
```

Comment:
Sorry would you be able to keep most of this in? 

Perhaps the readme.md should be really clear what the 
- OCF model does 
- Tryolabs model does

So hopefully it would just be adding a section for tryolabs and maybe refactoring

Reply:
I changed the readme and kept everything OCF specific inside. I added a passage about the tryolabs model in the Model and the Evaluation sections. Additionally, I added 2 points to the Known Restrictions section.